src/Asv.Mavlink.Shell/Commands/DownloadMissionElements.cs

```diff
@@ -0,0 +1,58 @@
+using System.Linq;
+using Asv.IO;
+using ConsoleAppFramework;
+using System.Threading;
+using Spectre.Console;
+
+namespace Asv.Mavlink.Shell;
+
+public class DownloadMissionElements
+{
+    /// <summary>
+    /// Selecting a device and uploading its missions
+    /// </summary>
+    /// <param name="connectionString">-cs, The address of the connection to the mavlink device</param>
+    /// <param name="iterations">-i, States how many iterations should the program work through</param>
+    /// <param name="devicesTimeout">-dt, (in seconds) States the lifetime of a mavlink device that shows no Heartbeat</param>
+    /// <param name="refreshRate">-r, (in ms) States how fast should the console be refreshed</param>
+    [Command("show-mission")]
+    public int RunShowMission(string connectionString = "tcp://127.0.0.1:5762", uint? iterations = null, uint devicesTimeout = 10, uint refreshRate = 3000)
```

Comment:
Тут не критично, но лучше длинные строчки кода превращать в более узкие при помощи переносов смысловых блоков.
```suggestion
public int RunShowMission(
        string connectionString = "tcp://127.0.0.1:5762", 
        uint? iterations = null, 
        uint devicesTimeout = 10, 
        uint refreshRate = 3000
)
```